redfish_utilities/logger.py

```diff
@@ -0,0 +1,55 @@
+#! /usr/bin/python
+# Copyright Notice:
+# Copyright 2019-2025 DMTF. All rights reserved.
+# License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/Redfish-Tacklebox/blob/main/LICENSE.md
+
+"""
+Logger
+
+File : logger.py
+
+Brief : Contains a unified logger that can be referenced from other files
+"""
+
+import logging
+import datetime
+import redfish
+import os
+
+
+def get_debug_level(level):
+    if level == "DEBUG":
+        return logging.DEBUG
+    elif level == "INFO":
+        return logging.INFO
+    elif level == "WARNING":
+        return logging.WARNING
+    elif level == "ERROR":
+        return logging.ERROR
+    elif level == "CRITICAL":
+        return logging.CRITICAL
+    else:
+        raise ValueError(f"Invalid debug level: {level}")
+
+
+def setup_logger(
```

Comment:
It looks like the `--debug` option isn't used when setting up the debug output. Can we add it in so that if specified, it logs to a file (and is set to "loggingDEBUG")?